decred/decred/dcr/dcrdata.py

```diff
@@ -889,20 +888,20 @@ def sendOutputs(self, outputs, keysource, utxosource, feeRate=None):
           (dcrwallet/wallet/txauthor).AddAllInputScripts
 
         Args:
-            outputs (list(TxOut)): The transaction outputs to send.
+            outputs list(TxOut): The transaction outputs to send.
             keysource func(str) -> PrivateKey: A function that returns the
                 private key for an address.
-            utxosource func(int, func(UTXO) -> bool) -> list(UTXO): A function
-                that takes an amount in atoms, and an optional filtering
-                function. utxosource returns a list of UTXOs that sum to >= the
-                amount. If the filtering function is provided, UTXOs for which
-                the  function return a falsey value will not be included in the
-                returned UTXO list.
+            utxosource func(int, func(UTXO) -> bool) -> (list(UTXO), bool):
+                A function that takes an amount in atoms, and an optional
+                filtering function. utxosource returns a list of UTXOs that sum
+                to >= the amount. If the filtering function is provided, UTXOs
+                for which the  function return a falsey value will not be
+                included in the returned UTXO list.
```

Comment:
Can you comment on the returned bool too?

Reply:
Probably because `updateTip` is called in `connect`, right after `DcrdataClient` creation, and if there's a problem connecting to the service, the latter is going to fail first.

Reply:
Indeed, thanks for the hint.